dfxdemo/dfxdemo.py

```diff
@@ -300,9 +300,40 @@ async def main(args):
         # Set the demographics
         if app.demographics is not None:
             print("    Setting user demographics:")
-            for k, v in app.demographics.items():
-                collector.setProperty(f"{k}:1", str(v))  # The :1 is because we only care about one face in this demo
-                print(f"       {k}: {v}")
+            if (gender := app.demographics.get("gender")) is not None:
+                sex_at_birth = dfxsdk.FaceValue.SEX_NOT_PROVIDED
+                if gender == "male":
+                    sex_at_birth = dfxsdk.FaceValue.SEX_ASSIGNED_MALE_AT_BIRTH
+                elif gender == "female":
+                    sex_at_birth = dfxsdk.FaceValue.SEX_ASSIGNED_FEMALE_AT_BIRTH
+                collector.setFaceAttribute("1", dfxsdk.FaceAttribute.SEX_ASSIGNED_AT_BIRTH, sex_at_birth)
+                print("       SEX_ASSIGNED_AT_BIRTH:", sex_at_birth)
+            if (age := app.demographics.get("age")) is not None:
+                collector.setFaceAttribute("1", dfxsdk.FaceAttribute.AGE_YEARS, age)
+                print(f"       AGE_YEARS: {age}")
+            if (height := app.demographics.get("height")) is not None:
+                collector.setFaceAttribute("1", dfxsdk.FaceAttribute.HEIGHT_CM, height)
+                print(f"       HEIGHT_CM: {height}")
+            if (weight := app.demographics.get("weight")) is not None:
+                collector.setFaceAttribute("1", dfxsdk.FaceAttribute.WEIGHT_KG, weight)
+                print(f"       WEIGHT_KG: {weight}")
+            if (smoking := app.demographics.get("smoking")) is not None:
+                collector.setFaceAttribute("1", dfxsdk.FaceAttribute.SMOKER, smoking)
+                print(f"       SMOKER: {smoking}")
+            if (diabetes_text := app.demographics.get("diabetes")) is not None:
+                diabetes = dfxsdk.FaceValue.DIABETES_NONE
```

Comment:
If a user enters invalid value for diabetes, it will become as "no diabetes" instead of diabetes answer not provided.